util/Hexagon/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.16)
+project(hexagon-instrument
+    VERSION 0.1
+    LANGUAGES CXX)
```

Comment:
Should this really be its own project?

Reply:
Not necessarily. What is a better place for it?

Reply:
Not necessarily. What is a better place for it?